### PR TITLE
Add Colette xPath scene, gallery scraper

### PR DESCRIPTION
**Pull request by bnkai**
as  requested from #73
<del>It is currently blocked as </del>
<del>- it needs a cookie to proceed to the scene/gallery page</del>
<del>- with the cookie set ( hardcoded for testing atm ) , the image seems to fail to download even though the url is legit</del>

### CODE REVIEW

**Comment by peolic** [comment]
> * with the cookie set ( hardcoded for testing atm ) , the image seems to fail to download even though the url is legit

Have you tried setting the cookies on the function that fetches the image from the scraped URL as well?

**Comment by bnkai** [comment]
@peolic it was a referrer issue 
using the cookies stash PR https://github.com/stashapp/stash/pull/934  and the referrer PR https://github.com/stashapp/stash/pull/968 combined seems to do the job.

**Comment by Belleyy** [comment]
You should write in `SCRAPERS-LIST.md` - **Needs**  
- a96ab9c (stash >= v0.4.0-14)

**Comment by bnkai** [comment]
> You should write in `SCRAPERS-LIST.md` - **Needs**
> 
>     * a96ab9c (stash >= v0.4.0-14)

lol i was just looking for the version number :-0

**Comment by bnkai** [comment]
@peolic @Belleyy i edited the scrapers list to remove stash versions needed older than the stable while i was fixing a merge conflict.

File: SCRAPERS-LIST.md [hunk 1]
```diff
@@ -124,17 +125,18 @@ bushybushy.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
...
 castingcouch-x.com|AMAMultimedia.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 charleechaselive.com|VNAGirls.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 cherrypimps.com|trafficpimps.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 chocolatebjs.com|PervCity.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 clips4sale.com|Clips4Sale.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 clubinfernodungeon.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-clubseventeen.com|ClubSeventeen.yml|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|stash >= v0.3.0-42|-
+clubseventeen.com|ClubSeventeen.yml|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|-|-
 collegefuckparties.com|Wtfpass.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+colette.com|Colette.yml|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|stash >= v0.4.0-14|-
```
Comment (peolic) [inline SCRAPERS-LIST.md:139]:
```suggestion
colette.com|Colette.yml|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|stash >= v0.4.0-14|-
collegefuckparties.com|Wtfpass.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```

File: scrapers/Colette.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      $info: //div[contains(@class,"columns info")]
+      $syn: //div[@class="cntr"]//div[@class=""]/h2
+    scene:
+      Title: //div[contains(@class,"row info")]//h1[not(@class)]
+      Details:
+        selector: $info//p
+        concat: "\n\n"
+        postProces:
+          - replace: "^HD\\sVideo:\\s*"
+            with:
```
Comment (peolic) [inline scrapers/Colette.yml:24]:
```suggestion
        postProcess:
          - replace:
              - regex: ^HD\sVideo:\s*
                with:
```

File: scrapers/Colette.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      $info: //div[contains(@class,"columns info")]
+      $syn: //div[@class="cntr"]//div[@class=""]/h2
+    gallery:
+      Title: //div[contains(@class,"row info")]//h1[not(@class)]
+      Details:
+        selector: $info//p
+        concat: "\n\n"
+        postProces:
+          - replace: "^HD\\sVideo:\\s*"
+            with:
```
Comment (peolic) [inline scrapers/Colette.yml:51]: postProcess:
          - replace:
              - regex: ^HD\sVideo:\s*
                with:

File: scrapers/Colette.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - colette.com/galleries
+    scraper: galleryScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[contains(@class,"columns info")]
+      $syn: //div[@class="cntr"]//div[@class=""]/h2
```
Comment (peolic) [inline scrapers/Colette.yml:16]: Unused? (gallery too)

File: scrapers/Colette.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        Name:
+          fixed: Colette
+      Date:
+        selector: ($info/h2)[1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+  galleryScraper:
+    common:
+      $info: //div[contains(@class,"columns info")]
+      $syn: //div[@class="cntr"]//div[@class=""]/h2
```
Comment (peolic) [inline scrapers/Colette.yml:43]: Unused?
```suggestion
```

**Comment by bnkai** [comment]
@peolic thanks i somehow used an older version of the scraper file

**Pull request merged**